x-pack/plugins/security_solution/public/detections/components/rules/related_integrations/utils.tsx

```diff
@@ -0,0 +1,113 @@
+/*
```

Comment:
Not a big fan of `utils.ts`, `types.ts`, and stuff like that. Could we split this into separate files?
1. One for `getIntegrationLink` (I'd make it a react FC called `RelatedIntegrationLink`)
2. And the other for `IntegrationDetails` and `getInstalledRelatedIntegrations` (`integration_details.ts`) - that one would actually contain our domain logic, so not "utils" please! 🙂

Reply:
Ended up being resolved in https://github.com/elastic/kibana/pull/134299